core/domain/exp_domain_test.py

```diff
@@ -7513,13 +7641,15 @@ class HTMLMigrationUnitTests(test_utils.GenericTestBase):
 objective: ''
 param_changes: []
 param_specs: {}
-schema_version: 38
+schema_version: 39
 states:
   Introduction:
     classifier_model_id: null
     content:
       content_id: content
-      html: ''
+      html: '<p>This is test </p><p><oppia-noninteractive-math math_content-with-value="{&amp;quot;raw_latex&amp;quot;:
+        &amp;quot;+,-,-,+&amp;quot;, &amp;quot;svg_filename&amp;quot;: &amp;quot;&amp;quot;}">
```

Comment:
Plan for adding proper value for svg_filane?

Reply:
As per discussion , not needed